Clean up the osu! World Cup #1 wiki article (#4731)
ppy/osu-wiki: wiki/Tournaments/OWC/1/en.md

![OWC #1 logo](img/logo.png)

The **osu! World Cup #1** (***OWC #1***) was a single elimination country-based osu! tournament that was being run by various osu! community members under the provision of the [osu! team](/wiki/People/The_Team). It was the first installment of the osu! World Cup.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2011-01-31/2011-02-21 |
| Round of 32 | 2011-02-22/2011-02-28 |
| Round of 16 | 2011-03-01/2011-03-07 |
| Quarterfinals | 2011-03-08/2011-03-22 |
| Semifinals | 2011-03-23/2011-03-29 |

> | 3rd Place Playoff & Finals | 2011-03-30/2011-04-11 |

Reviewer: ```suggestion
| Finals | 2011-03-30/2011-04-11 |
```

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter tag for each team member, unique profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu!supporter tag for each team member |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag for each team member |

![](img/badge.jpg "OWC #1 winner badge")

> Aside from the main prizes listed above, the Tournament Management also decided to separately award all members of the ![][flag_JP] Japanese team 1 month of osu!supporter tag each in the wake of the [2011 Tōhoku earthquake](https://en.wikipedia.org/wiki/2011_T%C5%8Dhoku_earthquake_and_tsunami) incident.

Reviewer: ```suggestion
Aside from the main prizes listed above, the tournament management has also decided to separately award all members of the ![][flag_JP] Japanese team with 1 month of osu!supporter tag in the wake of the [2011 Tōhoku earthquake](https://en.wikipedia.org/wiki/2011_T%C5%8Dhoku_earthquake_and_tsunami) incident.
```

## Organization

The osu! World Cup #1 was run by various community members.

| Position | Member(s) |
| :-: | :-- |
| Host/Manager | ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377) |
| Streamer | ![][flag_JP] [dvorak](https://osu.ppy.sh/users/271359) |

> | Map Selector | ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377), ![][flag_AU] [m980](https://osu.ppy.sh/users/3288), ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328), ![][flag_TW] [Alace](https://osu.ppy.sh/users/25993) |

Reviewer: ```suggestion
| Mappool selector | ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377), ![][flag_AU] [m980](https://osu.ppy.sh/users/3288), ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328), ![][flag_TW] [Alace](https://osu.ppy.sh/users/25993) |
```
